src/Ethereum/Transaction.cpp

```diff
@@ -66,7 +66,7 @@ Data TransactionNonTyped::serialize(const uint256_t chainID) const {
     return RLP::encodeList(encoded);
 }
 
-Data TransactionNonTyped::encoded(const Signature& signature, const uint256_t chainID) const {
+Data TransactionNonTyped::encoded(const Signature& signature, [[maybe_unused]] const uint256_t chainID) const {
```

Comment:
@catenocrypt looks good to remove to me

Reply:
`TransactionNonTyped` and `TransactionEip1559` inherits from the same interface, `TransactionEip1559` still use `chainID` I suggest to let the current behaviour.